ci/run_clang_tidy.sh

```diff
@@ -47,11 +54,12 @@ function filter_excludes() {
   exclude_testdata | exclude_chromium_url | exclude_win32_impl
 }
 
-LLVM_PREFIX=$(llvm-config --prefix)
 
 if [[ "${RUN_FULL_CLANG_TIDY}" == 1 ]]; then
   echo "Running full clang-tidy..."
-  "${LLVM_PREFIX}/share/clang/run-clang-tidy.py"
+  "${LLVM_PREFIX}/share/clang/run-clang-tidy.py" \
+    -clang-tidy-binary=${CLANG_TIDY} \
+    -clang-apply-replacements-binary=${CLANG_APPLY_REPLACEMENTS}
```

Comment:
You don't need this unless running with `fix`, and this script doesn't allow that either.

Reply:
That's true, but it's harmless and when I patch in the fix args it saves me patching that in too :)

Reply:
Can you make this script allow that too? :)

Reply:
Fixed.